.studio/common

```diff
@@ -31,8 +31,23 @@ function bootstrap_proto_compilers() {
   compile_go_protobuf components/automate-grpc/scripts/grpc.sh
 }
 
-# TODO (tc) Please take a pass @afiune and Infra Auto team to see if some of these other helpers
-# can be deleted in the new world.
+document "install_protoc_toolchain" <<DOC
+  Install all the protobuf plugins, linters, etc. we use to compile and vet our protobufs.
+DOC
+function install_protoc_toolchain() {
+  local proto_go_tools=(
+    github.com/chef/automate/components/automate-grpc/protoc-gen-policy
+    github.com/chef/automate/components/automate-grpc/protoc-gen-grpc-mock # generate mock grpc server implementations
+    github.com/chef/automate/components/automate-grpc/protoc-gen-a2-config
+    github.com/envoyproxy/protoc-gen-validate
+    github.com/ckaznocha/protoc-gen-lint
```

Comment:
I believe we can ditch this one if we're hooking up buf.

It's called here: https://github.com/chef/automate/blob/c9a5ab05e76c4fc4e91f0342caa5a16c1a431fb1/components/automate-gateway/scripts/grpc.sh#L86-L99

and we can remove the `nolint` comments from here, I suppose:
```console
$ git grep lint | grep proto:
api/config/gateway/config_request.proto:3:// nolint: 'V1' and 'S3' doesn't look like CamelCase to the linter. Curious.
components/automate-gateway/api/notifications/notifications.proto:3:// nolint: keeping this as-is for backwards-compat reasons (for now)
```

Reply:
👍 thanks for digging in, I'll put this on my list for when I start actually running buf against the codebase